### ISSUE TITLE
Catch wrong syntax for raising exceptions with `--py3k`

### ISSUE THREAD

**Issue by PCManticore** [PyCQA/pylint#1937]
### Steps to reproduce
1. Save the following into a file and lint it
```
raise Exception, "a", None
```

### Current behavior
We don't emit anything

### Expected behavior
We should emit that this syntax is invalid in Python 3

### pylint --version output
Pylint 1.8.X
Python 2

**Comment by kczapla** [comment]
Hey,
I tired to reproduce but I am getting:

> E:  1, 0: invalid syntax (<string>, line 1) (syntax-error)

Is it fixed already?

# pylint --version output:

__main__.py 2.0.0
astroid 1.5.3

**Comment by PCManticore** [comment]
Thanks for looking into this @kczapla ! Not quite, I assume you ran pylint with Python 3. We should warn about this syntax when we run pylint on Python 2 with the `--py3k` flag activated, for which this syntax is still valid.

**Comment by PCManticore** [comment]
Fixed in 1.9
